src/components/navigation/www/accountnav.scss

```diff
@@ -0,0 +1,91 @@
+@import "../../../colors";
+@import "../../../frameless";
+
+#account-nav {
```

Comment:
Please scope with a class rather than an id. In this case it's weird because there will practically only be one nav on a page, but since these are components, we shouldn't assume that.  Eventually we should try to get rid of all the id selectors in this repo.

Reply:
Also it looks like we're still styling some things in `navigation.scss` using `.account-nav`, so those would either need to be updated or removed.

Reply:
Re-did with class="account-nav" instead. Removed redundant lines from navigation.scss.